source/crud/write-operations/delete.txt

```diff
@@ -147,14 +148,47 @@ collection:
     { "_id": 1, "color": "red", "qty": 5 }
     { "_id": 8, "color": "black", "qty": 8 }
 
-For more information about the methods and classes mentioned in this guide,
-see the following resources:
-
-- `deleteOne() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteOne(org.bson.conversions.Bson)>`__ API Documentation 
-- `deleteMany() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteMany(org.bson.conversions.Bson)>`__ API Documentation
-- `findOneAndDelete() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#findOneAndDelete(org.bson.conversions.Bson)>`__ API Documentation
-- `DeleteOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/DeleteOptions.html>`__ API Documentation
-- `FindOneAndDeleteOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/FindOneAndDeleteOptions.html>`__ API Documentation
-- :manual:`db.collection.deleteOne() </reference/method/db.collection.deleteMany/>` Server Manual Entry
-- :manual:`db.collection.deleteMany() </reference/method/db.collection.deleteOne/>` Server Manual Entry
-- :manual:`db.collection.findOneAndDelete() </reference/method/db.collection.findOneAndDelete/>` Server Manual Entry
+Delete Example: Full File
+-------------------------
+
+The following code is a complete, standalone file that performs a delete one
+operation and a delete many operation. 
+
+.. include:: /includes/crud/example-intro.rst
+
+The query uses the ``eq()`` and ``lt()`` filters to query documents. For more
```

Comment:
S: same suggestion about keeping the code example description together and moving the /includes/crud/example-intro.rst

Reply:
I'd also move the link to the Filters API docs to a tip admonition after the code example